src/bors/merge_queue.rs

```diff
@@ -62,110 +62,49 @@ pub async fn merge_queue_tick(ctx: Arc<BorsContext>) -> anyhow::Result<()> {
         ctx.repositories.read().unwrap().values().cloned().collect();
 
     for repo in repos {
-        let repo_name = repo.repository();
-        let repo_db = match ctx.db.repo_db(repo_name).await? {
-            Some(repo) => repo,
-            None => {
-                tracing::error!("Repository {repo_name} not found");
-                continue;
-            }
-        };
+        let repo_name = repo.repository().to_string();
+        if let Err(error) = process_repository(repo, &ctx).await {
+            tracing::error!("Error processing repository {}: {:?}", repo_name, error);
```

Comment:
```suggestion
            tracing::error!("Error processing repository {repo_name}: {error:?}");
```
Clippy would soon warn about this :)